tcms/kiwi_auth/admin.py

```diff
@@ -9,6 +9,7 @@
 from django.http import HttpResponseForbidden, HttpResponseRedirect
 from django.urls import reverse
 from django.utils.translation import gettext_lazy as _
+from tcms.kiwi_auth.forms import GroupAdminForm
```

Comment:
IMO this form can live in the admin.py file b/c it's used only here and there's already a precedent with MyUserChangeForm (also in other admin.py files).